MediaGalleryUi/Model/SearchCriteria/CollectionProcessor/FilterProcessor/EntityType.php

```diff
@@ -65,4 +78,21 @@ private function getSelectByEntityType(array $value): Select
             $value
         );
     }
+    
+    /**
+     * Return select asset ids that not exists in asset_content_table
+     */
+    private function getNotUsedEntityIds(): Select
+    {
+        return $this->connection->getConnection()->select()->from(
+            ['mga' => $this->connection->getTableName(self::TABLE_MEDIA_GALLERY_ASSET)],
```

Comment:
```suggestion
            ['media_gallery_asset' => $this->connection->getTableName(self::TABLE_MEDIA_GALLERY_ASSET)],
```
Just be consistent with the subselect below